actionpack/test/dispatch/routing/inspector_test.rb

```diff
@@ -358,7 +383,7 @@ def test_no_routes_matched_filter
       end
 
       def test_no_routes_were_defined
-        output = draw(:filter => 'Rails::DummyController') { }
+        output = draw(defaults_filter: defaults_filter_for('post'), filter: 'post') {}
```

Comment:
It feels off that these tests change. We should do everything we can to make sure they don't have to change.